test/jdk/javax/swing/text/DefaultCaret/HidingSelection/HidingSelectionTest.java

```diff
@@ -89,7 +87,7 @@ public static void main(String[] args) throws Exception {
         robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
         robot.waitForIdle();
         robot.delay(200);
-        if (!biEqual(robot.createScreenCapture(bounds), sel)) {
+        if (!field2.getCaret().isSelectionVisible()) {
```

Comment:
This and the next test now checks behavior via the simple getter, but previously the test validated that the selection is actually visible on the screen which is more correct.